languages/common-lisp/exercises/concept/basics/.docs/instructions.md

```diff
@@ -1,17 +1,53 @@
-In this exercise, you fill need to fill in a number of incomplete functions.
+You! Yes you, brave lisper! [Lenny the lisp alien][alien] needs your help! He
```

Comment:
I love this new theme! Excellent work!

Reply:
The format of this `instructions.md` file is slightly different from the format used by the other tracks. In particular, we'd like to split things up explicitly into "tasks", each of which will involve doing one thing (define a function, implement a function, etc.). You can find an example of the format [here](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/instructions.md).

Splitting the instructions up in numbered tasks will allow us to add hints specific to individual tasks. See [this example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/hints.md).

Reply:
I think I've split things up a bit better now? I still have two functions in each task, but they are more task related than concept related groupings.

Reply:
Yep this is great! I've left some small comments.